test/unit/math/rev/mat/prob/multi_normal_cholesky_test.cpp

```diff
@@ -1,6 +1,12 @@
+#include <stan/math/prim/mat/fun/cholesky_decompose.hpp>
 #include <stan/math/rev/mat.hpp>
 #include <gtest/gtest.h>
 #include <test/unit/math/rev/mat/util.hpp>
+// For speed comparisons
```

Comment:
Commented out code should just be removed.

Reply:
the benchmarks were very useful for me, but I can kill these as well. These tests did identify quickly for me the problem with mdivide_right_tri.